wandb/integration/keras/callbacks/metrics_logger.py

```diff
@@ -0,0 +1,79 @@
+import sys
+from typing import Any, Dict, Optional, Union
+
+from tensorflow.keras import callbacks  # type: ignore
+
+import wandb
+from wandb.integration.keras.keras import patch_tf_keras
+from wandb.sdk.lib import telemetry
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+LOG_STRATEGY = Literal["epoch", "batch"]
+
+
+patch_tf_keras()
+
+
+class WandbMetricsLogger(callbacks.Callback):
+    """`WandbMetricsLogger` automatically logs the `logs` dictionary
+    that callback methods take as argument to wandb.
+
+    It also logs the system metrics to wandb.
+
+    Arguments:
+        log_freq ("epoch", "batch", or int): if "epoch", logs metrics
```

Comment:
I think this is the standard convention in the Keras callback world. See for example:
- https://keras.io/api/callbacks/tensorboard/ (search for `update_freq`)
- https://keras.io/api/callbacks/model_checkpoint/ (search for `save_freq`)